src/mote/templates/mainpage.html

```diff
@@ -33,6 +33,7 @@
         <link rel="icon" href="{{ url_for('static', filename='imgs/logoteal.png') }}" sizes="16x16">
         <script src="{{ url_for('static', filename='jscn/fragment.js') }}"></script>
         <script src="{{ url_for('static', filename='jscn/easy.qrcode.min.js') }}"></script>
+        <script src="https://cdnjs.cloudflare.com/ajax/libs/socket.io/4.3.2/socket.io.js"></script>
```

Comment:
As a reply to https://github.com/fedora-infra/mote/pull/201#issuecomment-982374808, the source is present in this [line](https://github.com/fedora-infra/mote/pull/201/files#diff-4c11e602a99bd1a8ca037ca1be41a6574c9c32ce26b2a7bee39fdd0c46fec471R36).

Reply:
@t0xic0der yes yes I got it thanks